### PR TITLE
Connected mailbox

### PR DESCRIPTION
**Pull request by Mirabellensaft**

### CODE REVIEW

File: exercise-solutions/connected-mailbox/simple_db/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[package]
+authors = ["Ferrous Systems"]
+edition = "2021"
```
Comment (jonathanpallant) [inline exercise-solutions/connected-mailbox/simple_db/Cargo.toml:3]: I added `edition = "2021"` to this example. We should do this to all our examples. The default edition is `2015` and in that edition you have to use `extern crate foo;` to use some crate called `foo`, which is annoying.

File: exercise-solutions/connected-mailbox/tcp-server/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[package]
+name = "tcp-server"
+version = "0.1.0"
+authors = ["Florian Gilcher <florian.gilcher@asquera.de>"]
```
Comment (jonathanpallant) [inline exercise-solutions/connected-mailbox/tcp-server/Cargo.toml:4]: We should set the authors to "Ferrous Systems" rather than individuals.

**Comment by jonathanpallant** [comment]
Wrote some help text.

**Comment by jonathanpallant** [comment]
Rebased

**Review by listochkin — APPROVED** [review]

**Pull request merged**